swap/src/monero.rs

```diff
@@ -99,6 +99,31 @@ impl Amount {
         self.0
     }
 
+    pub fn as_monero_decimal(&self) -> Decimal {
```

Comment:
Do we still need this now?

Reply:
not really, do I remove it or leave it be?

Reply:
Remove it then please!

In general, I'd prefer if we strive towards having only code around that is actively used and relevant :)

Reply:
For PRs in particular, it is easier to review if the diff is minimal.

Reply:
Done!